scraper/src/tests/conftest.py

```diff
@@ -0,0 +1,4 @@
+import os
```

Comment:
To allow to override the environnement. Otherwise your test will fail if you don't set the API_KEY and APPLICATION_ID in your shell.